libsvmdata/__init__.py

```diff
@@ -1,5 +1,60 @@
-from .datasets import fetch_libsvm, download_libsvm, NAMES  # noqa
```

Comment:
we cannot remove existing functions without deprecating them first, otherwise it will break outside codebases that rely on it

Reply:
Do you think I should re-add the `dataset.py` file or rather to add new `fetch_libsvm`, `download_libsvm` and `NAMES` functions/values that use the new functions in my PR ?

Reply:
The 2nd option: In libsvm'.dataset I'd expose the two aforementioned function, that call the new functions and raise a FutureWarning about deprecation in v0.5 and removal in v0.6
+ import them in the init so that they remain importable as before